tests/browser/features/language-toggle.feature

```diff
@@ -0,0 +1,34 @@
+@mock-api:session-500 @lang
+Feature: Error handling
```

Comment:
Ah actually - this feature probably needs to be changed to something more relevant?

Reply:
I'll get that done now